test/fpm_test/test_manifest.f90

```diff
@@ -1407,8 +1410,11 @@ subroutine test_macro_parsing_dependency(error)
 
         if (allocated(error)) return
 
-        macrosPackage = get_macros(id, package%preprocess(1)%macros, package%version%s())
-        macrosDependency = get_macros(id, dependency%preprocess(1)%macros, dependency%version%s())
+        pkg_ver = package%version%s()
+        dep_ver = dependency%version%s()
+
+        macrosPackage = get_macros(id, package%preprocess(1)%macros, pkg_ver)
```

Comment:
I don't really like the `camelCase` usage here but ofcourse that is out of the scope of this PR.